src/main/java/com/maddyhome/idea/vim/group/EditorGroup.java

```diff
@@ -257,6 +255,14 @@ public void notifyIdeaJoin(@NotNull VimEditor editor) {
     notifyIdeaJoin(((IjVimEditor) editor).getEditor().getProject());
   }
 
+  @NotNull
+  @Override
+  public Collection<VimEditor> getAllEditors() {
```

Comment:
Why don't you use `localEditors`? I suppose the method you've created may return remote editors in case of CWM.